employee-rostering-frontend/pom.xml

```diff
@@ -28,6 +28,7 @@
   <artifactId>employee-rostering-frontend</artifactId>
   <name>Employee Rostering as a Service React Frontend</name>
   <description>React frontend for Employee Rostering as a Service</description>
+  <packaging>pom</packaging>
```

Comment:
This won't work. You need some kind of archive (jar, war, zip) you can consume in the standalone module.

Reply:
Couldn't that be done with a plugin though? (And if I recall correctly, there no built-in packaging type for zip). (Probably will readd the assembly plugin, and make it assemble a zip of build).

Reply:
Used a plugin to add a Zip artifact, which appears to work

Reply:
OK, looks good. Please remove the resource plugin, it's not needed anymore.